resources/assets/components/FilterBar/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './filter-bar.scss';
+
+import DropdownFilter from '../DropdownFilter';
```

Comment:
Don't think you need these anymore since they are used in `CampaignSingle`

Reply:
oh right!